frontend/apps/web/yup-validations/connections.ts

```diff
@@ -76,23 +76,30 @@ const connectionNameSchema = Yup.string()
 export const SshTunnelFormValues = Yup.object({
   host: Yup.string(),
   port: Yup.number()
-    .min(0)
-    .when('host', (host, schema) => ([host] ? schema.required() : schema)),
+    .min(0, 'The Port must be greater than 0.')
```

Comment:
not sure this message is quite right, we allow the minimum to be 0. So it should be greater than or equal to.